src/effects/backends/builtin/pitchshifteffect.cpp

```diff
@@ -0,0 +1,98 @@
+#include "effects/backends/builtin/pitchshifteffect.h"
+
+#include "util/sample.h"
+
+PitchShiftGroupState::~PitchShiftGroupState() noexcept {
+    SampleUtil::free(m_retrieveBuffer[0]);
+    SampleUtil::free(m_retrieveBuffer[1]);
+
+    VERIFY_OR_DEBUG_ASSERT(m_pRubberBand) {
+        return;
+    }
+
+    m_pRubberBand->reset();
+}
+
+// static
+QString PitchShiftEffect::getId() {
+    return "org.mixxx.effects.pitchshift";
```

Comment:
This could become a constant.

Reply:
And also a `QLatin1String`

Reply:
I guess the result is expected to be a `QString` so using `QLatin1String` would not make sense here.

Reply:
Mhmm I figured it was faster but looking at the Qt implemenation, you are right its not. We should change that in the future.